Task5/database/connection.go

```diff
@@ -0,0 +1,26 @@
+package database
+
+import (
+	"database/sql"
+
+	_ "github.com/lib/pq"
+)
+
+const URL = "postgres://bacancy:admin@localhost/bacancy?sslmode=disable"
```

Comment:
@ketan-rathod-713 

Make this string to dynamic and use env variables.

Use Enviroment variable for the database creds, database details.

Reply:
Used godotenv package to load environment variables from .env file in given directory.